### PR TITLE
Tiny bug fix, update for new format, and gradle

### PR DESCRIPTION
**Pull request by b0bam**
Bug mentioned was that when you scrolled while mouse was hovering on
chat gui, it would recognise it as a click.

### CODE REVIEW

File: src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
 public class ChatUtil
 {
     private static final Pattern CHANNEL_PATTERN = Pattern
-            .compile("(\\[(G|W|H|T|L|F|S|P|A|PM){1}\u25AA?(B1|B2|B3|B4|S|G|N|M|MG|PvP)?\\]){1}");
+            .compile("(\\[(G|W|H|T|L|F|S|P|A|PM|\u0053\u0070\u0079){1}\u25AA?(B1|B2|B3|B4|S|G|N|M|MG|PvP|SP)?\\]){1}");
```
Comment (Fridgey) [inline src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java]: Confused as to why you're spelling "spy" in unicode.

Comment (b0bam) [inline src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java]: To confuse you

File: src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java [hunk 2]
```diff
@@ -58,13 +67,13 @@
...
-        String[] firstParts = message.substring(0, message.indexOf(" ")).split("=");
+        String[] firstParts = message.substring(0, message.indexOf(" ")).split("\\|");
         String chat = message.substring(message.indexOf(" "));
 
         if (firstParts.length < 3)
         {
             return message;
         }
-        return firstParts[0] + "=" + firstParts[firstParts.length - 1] + chat;
+		return firstParts[0] + "|" + firstParts[firstParts.length - 1] + chat;
```
Comment (Fridgey) [inline src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java]: HOW DARE YOU ADD AN EXTRA TAB!! I SHALL BEAT YOU WITH A SHRUBBERY!

File: src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java [hunk 3]
```diff
@@ -19,18 +19,27 @@
         {
             return Optional.absent();
         }
-        String[] firstParts = noColorMessage.substring(0, noColorMessage.indexOf(" ")).split("=");
+        String delimiter = "";
```
Comment (Fridgey) [inline src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java]: Why not just change the "=" to a "\\|"? Unless both are being used, which I don't think is the case.

File: src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java (hunk 2 above)
Comment (b0bam) [inline src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java]: But mom, I probably had "if's" and stuff which I then deleted

File: src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java (hunk 3 above)
Comment (b0bam) [inline src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java]: alt text

Comment (b0bam) [inline src/main/java/io/github/fridgey/chatmacros/util/ChatUtil.java]: 73109f6513ccdd2f53f7ca209a6673a8bd41328f

**Pull request merged**